src/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
```

Comment:
```suggestion
import React from 'react';
import ReactDOM from 'react-dom';

import App from './App';
```

리액트 라이브러리 임포트와 로컬 컴포넌트 임포트를 구분하기 위해한 줄 공백을 주시면 좋을 것 같습니다!
지금은 모듈이 별로 없지만, 모듈 간에도 역할이 있고, 분리를 해두는 습관을 두면 좋을 것 같아요

다른 파일들도 적용해주세요~